src/errors/error.ts

```diff
@@ -0,0 +1,81 @@
+import { ZodIssue } from 'zod';
+import { MongooseError } from 'mongoose';
+
+export interface BadIssue {
```

Comment:
Not sure if there are issues that are not bad :smile: 
```suggestion
export interface Issue {
```